scripts/ci-build-azure-ccm.sh

```diff
@@ -48,7 +48,9 @@ setup() {
 
     # the azure-cloud-provider repo expects IMAGE_REGISTRY.
     export IMAGE_REGISTRY=${REGISTRY}
-    pushd "${AZURE_CLOUD_PROVIDER_ROOT}" && IMAGE_TAG=$(git rev-parse --short=7 HEAD) && export IMAGE_TAG && popd
+    pushd "${AZURE_CLOUD_PROVIDER_ROOT}" && IMAGE_TAG=$(git rev-parse --short=7 HEAD) &&
+      IMAGE_TAG_CCM="${IMAGE_TAG}" && IMAGE_TAG_CNM="${IMAGE_TAG}" &&
+      export IMAGE_TAG && export IMAGE_TAG_CCM && export IMAGE_TAG_CNM && popd
```

Comment:
here we can simply do 

```
 IMAGE_TAG_CCM="${IMAGE_TAG_CCM:-IMAGE_TAG}" && IMAGE_TAG_CNM="${IMAGE_TAG_CNM:-IMAGE_TAG}" &&
      export IMAGE_TAG_CCM && export IMAGE_TAG_CNM && popd
```

This will ensure `IMAGE_TAG_CNM` and `IMAGE_TAG_CCM` get defaulted to the value of `IMAGE_TAG` if they are not set, but only if not already set.

Reply:
Updated.